app/src/main/java/com/eventyay/organizer/core/main/MainActivity.java

```diff
@@ -141,7 +146,17 @@ public void onDrawerClosed(View drawerView) {
     public void setEventId(long eventId) {
         this.eventId = eventId;
         fragmentNavigator.setEventId(eventId);
-        binding.navView.getMenu().setGroupVisible(R.id.subMenu, true);
+        binding.navView.getMenu().findItem(R.id.nav_dashboard).setVisible(true);
+        binding.navView.getMenu().findItem(R.id.nav_attendees).setVisible(true);
+        binding.navView.getMenu().findItem(R.id.nav_share).setVisible(true);
+        binding.navView.getMenu().findItem(R.id.nav_about_event).setVisible(true);
+        binding.navView.getMenu().findItem(R.id.nav_event_settings).setVisible(true);
```

Comment:
Instead, create a list of normal features and iterate

Reply:
Done.